WiserTaskScheduler/WiserTaskScheduler/Core/Interfaces/IParentUpdateService.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Threading.Tasks;
+using WiserTaskScheduler.Core.Models;
+
+namespace WiserTaskScheduler.Core.Interfaces
+{
+    /// <summary>
+    /// A service to perform updates to parents whose child items got updated.
+    /// </summary>
+    public interface IParentUpdateService
+    {
+        /// <summary>
+        /// Gets or sets the log settings that the Main service needs to use.
```

Comment:
Comment refers to the Main service while this is the Parent update service.

Reply:
done